Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    decanter (3.4.2)
+    decanter (3.4.3)
```

Comment:
I would consider this a minor release (i.e., `3.5.0`) since it's adding new, backwards compatible functionality